MAUI/NavigationDrawer/Getting-Started.md

```diff
@@ -0,0 +1,332 @@
+
+This section explains the steps required to configure the [.NET MAUI NavigationDrawer]() with a content area and a data-filled drawer. It covers only the minimal features that you need to know to get started with the NavigationDrawer.
+
+## Adding a .NET MAUI SfNavigationDrawer reference
+
+Syncfusion .NET MAUI controls are available in [Nuget.org](https://www.nuget.org/). To add [.NET MAUI NavigationDrawer]() to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.NavigationDrawer]() and then install it.
+
+## Handler registration 
+
+ In the MauiProgram.cs file, register the handler for Syncfusion core.
+
+{% highlight c# hl_lines="6 17" %}
+using Microsoft.Maui;
+using Microsoft.Maui.Hosting;
+using Microsoft.Maui.Controls.Compatibility;
+using Microsoft.Maui.Controls.Hosting;
+using Microsoft.Maui.Controls.Xaml;
+using Syncfusion.Maui.Core.Hosting;
+
+namespace NavigationDrawerSample
+{
+    public static class MauiProgram
+    {
+    public static MauiApp CreateMauiApp()
+    {
+        var builder = MauiApp.CreateBuilder();
+        builder
+        .UseMauiApp<App>()
+        .ConfigureSyncfusionCore() 
+        .ConfigureFonts(fonts =>
+        {
+            fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+        });
+
+        return builder.Build();
+        }      
+    }
+}     
+
+{% endhighlight %} 
+
+## Create a Simple .NET MAUI NaviagtionDrawer
+
+The .NET MAUI NaviagtionDrawer control is configured entirely in C# code or by using XAML markup. The following steps explain how to create a .NET MAUI NaviagtionDrawer (SfNaviagtionDrawer) and configure its elements:
+
+### Adding the .NET MAUI NaviagtionDrawer control
+
+Step 1: Add the NuGet to the project as discussed in the above reference section. 
+
+Step 2: Add the namespace as shown in the following code sample:
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    xmlns:navigationdrawer="clr-namespace:Syncfusion.Maui.NavigationDrawer;assembly=Syncfusion.Maui.NavigationDrawer"
+	
+{% endhighlight %}
+
+{% highlight c# %}
+
+    using Syncfusion.Maui.NavigationDrawer;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+Step 3: Set the control to content in `ContentPage.`
+
+{% tabs %}
+
+{% highlight xaml %}
+
+
+<navigationdrawer:SfNavigationDrawer x:Name="navigationDrawer">
+    <navigationdrawer:SfNavigationDrawer.ContentView>
+        <Grid/>
+    </navigationdrawer:SfNavigationDrawer.ContentView>
+</navigationdrawer:SfNavigationDrawer>
+
+
+{% endhighlight %}
+
+{% highlight c# %}
+          
+SfNavigationDrawer navigationDrawer = new SfNavigationDrawer();
+Grid grid = new Grid();
+navigationDrawer.ContentView = grid;
+this.Content = navigationDrawer; 
+
+{% endhighlight %}
+
+{% endtabs %}
+
+N> It is mandatory to set [`ContentView`]() for [`SfNavigationDrawer`]() on initializing.
+
+## Adjust Drawer Size
+
+The default position of the navigation pane is on the left, so let's change the drawer width to 250.
```

Comment:
Highlight the API names,
The default position of the navigation pane is on the left, so let's change the drawer width to 250 by using the `DrawerWidth` property.